objects/obj_p_assra/Step_0.gml

```diff
@@ -89,9 +89,13 @@ if (boarding=true) and (board_cooldown>=0) and (instance_exists(target)) and (in
                 // Bonuses
                 difficulty+=unit.experience/20;
                 difficulty+=(1-(target.hp/target.maxhp))*33;
-                //TODO define tag for bording weapons
-                if (array_contains(["Chainfist","Meltagun","Lascutter","Boarding Shield"], unit.weapon_one())) then difficulty+=3;
-                if (array_contains(["Chainfist","Meltagun","Lascutter","Boarding Shield"], unit.weapon_two())) then difficulty+=3;
+                //tag for boarding weapons = boarding 1-3
+                if (array_contains(["Chainsword","Power Sword","Force Sword","Bolter","Multi-Melta","Plasma Pistol","Plasma Gun","Bolt Pistol"], unit.weapon_one())) then difficulty+=2;
```

Comment:
Previously, you've also suggested:
> Consider refactoring the array_contains checks to use the 'boarding X' weapon tags instead of hardcoded weapon lists. This would be more maintainable and consistent with the weapon data structure.

Can you provide a code suggestion for it as well?

Reply:
what is `includes()`? I never saw that in gamemaker.